check_bbox/test.py

```diff
@@ -0,0 +1,27 @@
+from signature_detect.loader import Loader
+from signature_detect.cropper import Cropper
+from signature_detect.extractor import Extractor
+from signature_detect.bounding_box import Bounding_box
+import cv2
+
+path = '/Users/veersingh/Desktop/docs_with_signs_dataset(tobacco800)/images/aam09c00.tif'
```

Comment:
The absolute path make the tests difficult on my side :(

Reply:
That is my bad. 
`path = 'aam09c00.tif'`
should work